.github/workflows/android_deploy.yml

```
@@ -38,10 +38,13 @@ jobs:
         env:
           FIREBASE_OPTIONS_BASE64: ${{ secrets.FIREBASE_OPTIONS_BASE64 }}
           GOOGLE_SERVICES_JSON_BASE64: ${{ secrets.GOOGLE_SERVICES_JSON_BASE64 }}
+          APP_SECRETS_BASE64: ${{ secrets.APP_SECRETS_BASE64 }}
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)